client/my-sites/media-library/list-item.scss

```diff
@@ -71,19 +74,22 @@
 .media-library__list-item.is-transient .media-library__list-item-figure::after {
 	content: '';
 	position: absolute;
-		top: 0;
-		right: 0;
-		bottom: 0;
-		left: 0;
+	top: 0;
+	right: 0;
```

Comment:
Just to note, the style guidelines do advocate the indentation here under the "Positioning" section. I wonder if that should be changed though, it's rare to see it for some reason.

Reply:
Yep, I think that's introduced by running prettier locally on those files. Let's see if stylelint can fix those.